backend/clubs/views.py

```diff
@@ -4899,36 +4902,36 @@ def update(self, *args, **kwargs):
         """
         Updates times for all applications with cycle
         """
-        applications = ClubApplication.objects.filter(
-            application_cycle=self.get_object()
-        )
-        str_start_date = self.request.data.get("start_date").replace("T", " ")
-        str_end_date = self.request.data.get("end_date").replace("T", " ")
-        str_release_date = self.request.data.get("release_date").replace("T", " ")
+        cycle = self.get_object()
+        applications = ClubApplication.objects.filter(application_cycle=cycle)
+
+        new_start = self.request.data.get("start_date").replace("T", " ")
+        new_end = self.request.data.get("end_date").replace("T", " ")
+        new_release = self.request.data.get("release_date").replace("T", " ")
```

Comment:
@julianweng you show up as the last author for these lines -- are these supposed to replace with spaces, or empty strings? Confused how the lines below (`if new_start`) ever evaluates to false if we're replacing with a non-empty string.